chains/orchestrator-relays/runtime/starlight/src/lib.rs

```diff
@@ -1670,11 +1670,7 @@ impl Contains<RuntimeCall> for MaintenanceFilter {
 }
 
 /// Normal Call Filter
-type NormalFilter = EverythingBut<(
-    IsRelayRegister,
-    IsParathreadRegistrar,
-    IsDemocracyExtrinsics,
-)>;
+type NormalFilter = EverythingBut<(IsRelayRegister, IsParathreadRegistrar)>;
```

Comment:
@girazoki , pay attention here; I've enabled democracy extrinsics.

Reply:
I would say it's ok to enable them, but let's discuss it